src/main/scala/com/github/tkawachi/doctest/ScaladocTestGenerator.scala

```diff
@@ -17,9 +17,8 @@ object ScaladocTestGenerator {
    * Generates test source code from scala source file.
    */
   def apply(srcFile: File, srcEncoding: String, testGen: TestGen, decodeHtmlEnabled: Boolean, onlyCodeBlocksMode: Boolean): Seq[TestSource] = {
-    val src = Source.fromFile(srcFile, srcEncoding).mkString
```

Comment:
I think `Source.fromFile(..).mkString` leaks file handles, it's better to use `java.nio.file.Files.readAllBytes` in my experience

Reply:
:+1: